src/plugins/github/heuristics/mentionsAuthorReference.test.js

```diff
@@ -136,6 +136,7 @@ describe("plugins/github/heuristics/mentionsAuthorReference", () => {
           login: "my-owner",
           url: "https://github.com/my-owner",
         },
+        defaultBranchRef: null,
```

Comment:
Please investigate why this had no Flow error previously. The parameter to `addData` has type `GithubResponseJSON`, which should have caught this.

Reply:
Filed: <https://github.com/gajus/eslint-plugin-flowtype/issues/353>